ui/app/adapters/clients/activity.js

```diff
@@ -46,6 +47,30 @@ export default class ActivityAdapter extends ApplicationAdapter {
     });
   }
 
+  async exportData(query) {
+    const url = `${this.buildURL()}/internal/counters/activity/export${queryParamString({
+      format: query?.format || 'csv',
+      start_time: query?.start_time ?? undefined,
+      end_time: query?.end_time ?? undefined,
+    })}`;
+    let errorMsg;
+    try {
+      const options = query?.namespace ? { namespace: query.namespace } : {};
+      const resp = await this.rawRequest(url, 'GET', options);
+      if (resp.status === 200) {
+        return resp.blob();
+      }
+      // If it's an empty response (eg 204), there's no data so return an error
+      errorMsg = 'no data to export in provided time range.';
```

Comment:
NIT: caps 'no' -> 'No'

Reply:
I'm matching the casing of the API error messages, which don't have capitalization